inspirehep/modules/workflows/static/js/inspire_workflows_ui/templates/results.html

```diff
@@ -73,10 +73,9 @@
         </div>
 
         <div class="col-md-2 pad-top-10">
-          <ul ng-init="categories = record._source.metadata.inspire_categories || record._source.metadata.arxiv_categories" class="subject-terms">
-            <li
-              ng-repeat="subject in categories | limitTo:3"
-              ng-if="record._source.metadata.inspire_categories || record._source.metadata.arxiv_categories">{{subject.term || subject}}
+          <ul class="subject-terms">
+            <li ng-repeat="category in record._source.metadata | categories | limitTo:3">
```

Comment:
In the old Holding Pen there seems to be no limit. I would remove it to avoid confusing people.